Add the Headway project to Maps/GPS

ellenhp opened the pull request:
<!-- DO NOT DELETE THE TEXT BELOW. Please make sure relevant boxes are checked [x] -->

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

nodiscc reviewed (changes requested):
The description could be improved. Based on official documentation can you please change the description to:

> Self-hostable maps stack, powered by OpenStreetMap - take your location data into your own hands. Over 200 different cities are currently supported.

ellenhp commented:
I was hoping I could convey the difference between Headway and other projects that focus only on static maps. Headway also has a geocoder, routing engine, transit trip planner, etc, all integrated into one UI. "Maps stack" doesn't really convey that. Do you need the description to match the repo description? Because I guess I could go change that too if need be :)

nodiscc commented:
> Do you need the description to match the repo description?

Not necessarily, I just didn't find the description very clear

> Headway also has a geocoder, routing engine, transit trip planner, etc, all integrated into one UI.

That makes it clearer.

I suggest changing the description here (and in your project README if you like) to

> Self-hostable maps stack, powered by OpenStreetMap, including geocoder, routing engine, transit trip planner, all integrated into one UI. Over 200 different cities are currently supported.

nodiscc commented:
@ellenhp ping

ellenhp commented:
Thank you for the reminder!!! I'll do my best to get to this soon :)

ellenhp commented:
Hey, sorry this took so long. I've updated the PR with the new demo URL and updated information based on the clearer example you gave. The scope of the project has expanded since I wrote this PR originally though so the 200+ cities thing isn't relevant anymore :)

nodiscc reviewed (approved):
thanks
